data/map altera.txt

```diff
@@ -4292,56 +4292,56 @@ system Zephyrius
 	shipyard "Ancient Basic"
 	outfitter "Ancient Advanced"
 
 planet Atlantis
 	landscape land/beach11-harro
-	description `Floating in the middle of the ocean is the city of Atlantis, an Alteran city ship, you are unsure whether it is still capable of flying, but it may very well be. Apart from the luxury oriented shipyard and outfitter, there isn't much manufacturing going on here, mining and agriculture can only be found on the few small landmasses on this world and even then, it is barely present.`
+	description `Floating in the middle of the ocean is the city of Atlantis, an Alteran city ship. You are unsure whether it is still capable of flying, but it may very well be. Apart from the luxury oriented shipyard and outfitter, there isn't much manufacturing going on here, mining and agriculture can only be found on the few small landmasses of this world and even there, they are barely present.`
 	description `	Considering everything, there is no question as to why this is called a paradise world, the city is highly limited in terms of living space, due to its nature as a city ship, and is most definitely shielded and capable of submerging, a defensive system used during the Drak-Alteran War, explaining the myth of the sunken city of Atlantis. It's one of the safest places in Alteran space, if you can afford the exorbitant housing prices.`
-	spaceport `The spaceport is placed around the city on its arms and appears to be well guarded, not only by the military personnel present, but also by the air support checking every ship before landing. Unlike your usual human spaceport, everything is highly organised here and almost eerily quiet, the personnel immediately take care of both yourself and your fleet.`
-	spaceport `	While the spaceport is not a place you feel comfortable staying in, the luxurious spaceport bar on the 3rd arm is a far nicer location, with a great view over the spaceport and surrounding ocean. The food and drinks hear leave nothing to be desired either.`
+	spaceport `The spaceport is placed around the city on its arms and appears to be well guarded, not only by the military personnel present, but also by the air support checking every ship before landing. Unlike your usual human spaceport, everything is highly organized here and almost eerily quiet, the personnel immediately take care of both yourself and your fleet.`
+	spaceport `	While the landing terminal is not a place you feel comfortable staying in, the luxurious spaceport bar on the 3rd arm is a far nicer location, offering great views of the spaceport and the surrounding ocean. The food and drinks here leave nothing to be desired.`
 	shipyard "Ancient Basic"
 	outfitter "Ancient Basic"
```

Comment:
As much as it pains me, because I also prefer writing British english, the game is written in US english. Therefore it is organized and not organised - I usually trust my word spell check there. 
The next sentence I tried to avoid having the word spaceport in it 3 times. But I am very unsure about this suggestion, because I don't know if it still expresses properly what you wanted to say.